CONTRIBUTING.md

```diff
@@ -12,80 +12,69 @@ This project is a regular [Kubernetes Operator](https://coreos.com/operators/)
 
 ### Installing the Operator SDK command line tool
 
-Follow the installation guidelines from [Operator SDK GitHub page](https://github.com/operator-framework/operator-sdk) or run `make install-sdk`.
+Follow the installation guidelines from [Operator SDK GitHub page](https://github.com/operator-framework/operator-sdk)
 
 ### Developing
 
 As usual for operators following the Operator SDK in recent versions, the dependencies are managed using [`go modules`](https://golang.org/doc/go1.11#modules). Refer to that project's documentation for instructions on how to add or update dependencies.
 
-The first step is to get a local Kubernetes instance up and running. The recommended approach is using `minikube`. Refer to the Kubernetes'  [documentation](https://kubernetes.io/docs/tasks/tools/install-minikube/) for instructions on how to install it.
+The first step is to get a local Kubernetes instance up and running. The recommended approach for development is using `minikube` with *ingress* enabled. Refer to the Kubernetes'  [documentation](https://kubernetes.io/docs/tasks/tools/install-minikube/) for instructions on how to install it.
 
 Once `minikube` is installed, it can be started with:
 
 ```
-minikube start
+minikube start --addons=ingress
 ```
 
 NOTE: Make sure to read the documentation to learn the performance switches that can be applied to your platform.
 
+Log into docker (or another image registry):
+```
+docker login --username <dockerusername>
+```
+
 Once minikube has finished starting, get the Operator running:
 
 ```
 make cert-manager
-IMG=docker.io/$USER/jaeger-operator:latest make generate bundle docker push deploy
+export IMG_PREFIX=docker.io/<dockerusername>
+export IMG=${IMG_PREFIX}/jaeger-operator:latest
```

Comment:
any reason why the command is split? The original one is a single like copy paste and run which is easier to do.